docs/faq/structure.md

```diff
@@ -1,32 +1,45 @@
 # File structure used and created by Ember-Electron
 
-The `ember-electron` folder created at the root of your project by `ember-electron`'s blueprint looks like this:
+The `ember-electron` folder created at the root of your project by
+`ember-electron`'s blueprint looks like this:
 
 ```
  ember-electron
  ├── .compilerc
  ├── .electron-forge
- ├── lib
- │   └── index.js
+ ├── index.js
  ├── resources
  ├── resources-darwin
  ├── resources-linux
  └── resources-win32
 ```
 
-The `lib` directory is meant for code, and the `resources` directories are meant for non-code resources, such as images, json files, etc. The `resources-<platform>` directories exist so you can supply resources that will only be included on certain platforms, augmenting or replacing your platform-independent resources in `resources`. When the `ember electron` assembles your Electron project, it will look like this:
+The `resources` directories are meant for non-code resources, such as images,
+json files, etc. The `resources-<platform>` directories exist so you can supply
+resources that will only be included on certain platforms, augmenting or
+replacing your platform-independent resources in `resources`. When the `ember
+electron` assembles your Electron project, it will look like this:
 
 ```
  ember-electron
- ├── .compilerc
  ├── package.json
- ├── .electron-forge
- ├── lib
- │   └── index.js
- ├── resources
- └── ember
-     ├── index.html
-     └── <the rest of your built ember app>
+ ├── .compilerc
+ ├── ember
+ │   ├── <ember build output>
+ ├── ember-electron
+ │   ├── .electron-forge
+ │   ├── index.js
```

Comment:
Yeah, I'd be fine with killing this whole comment and just pointing readers to the README or documentation site or whatever if you'd rather do that than update this comment.